.gitmodules

```diff
@@ -1,3 +1,3 @@
 [submodule "assets/src/styles/styleguide"]
```

Comment:
Preserving this name will probably make it easier to pull in the changes, since it just needs to update the remote and path.

Reply:
Github seems to conflate the folder name and the submodule name :confused: 
![Screenshot from 2021-03-26 11-13-58](https://user-images.githubusercontent.com/7604138/112616811-74887980-8e24-11eb-8889-7bc5ee274438.png)

Reply:
So probably `[submodule "assets/src/styles/styleguide"]` is just a unique identifier within `.gitmodules`.